docs/classic-vs-vt.md

```diff
@@ -0,0 +1,81 @@
+
+As Windows has evolved, the security controls and restrictions on window handles have increased. Additionally, the nature and existence of an application-addressable window handle on any specific user interface element has evolved, especially with the increased support of device form factors and platforms. This makes direct window access to command-line applications fragile as the platform and experiences evolve.
+
+### Unicode
+
+The implicit standard for communication across platforms and the web is Unicode, specifically in the UTF-8 form. Of course, other encodings still exist, but generally speaking, when otherwise undefined, using UTF-8 is accepted as the appropriate default balance of portability, storage/transmission size, and breadth of expression required to support the world's languages and glyphs.
+
+The Windows console platform has and will continue to support all existing code pages and encodings, but we recommend UTF-8 for all forward looking development and also accept UTF-16 as an algorithmically-translatable alternative representation of the same information that is more compatible with Windows' historical Unicode philosophies extending from initial UCS-2 implementations.
+
+UTF-8 support in the console can be found via the _A_ variant of all Console APIs or the filesystem [**WriteFile**](https://msdn.microsoft.com/library/windows/desktop/aa365747) and [**ReadFile**](https://msdn.microsoft.com/library/windows/desktop/aa365467) methods against console handles after setting the codepage to `65001` or `CP_UTF8` with the [**SetConsoleOutputCP**](setconsoleoutputcp.md) and [**SetConsoleCP**](setconsolecp.md) methods as appropriate. Setting the code pages in advance is only necessary if the machine has not chosen "Use Unicode UTF-8 for worldwide language support" in the settings for Non-Unicode applications in the Region section of the Control Panel.
+
+UTF-16 support in the console can be found with no additional configuration via the _W_ variant of all console APIs and is a more likely choice for applications already well versed in UTF-16 through communication with the `wchar_t` and _W_ variant of other Microsoft and Windows platform functions and products.
+
+## Recommendations
+
+For all new and ongoing development on Windows, virtual terminal sequences are recommended as the way of interacting with the terminal. This will converge Windows command-line client applications with the style of application programming on all other platforms.
+
+A limited subset of Windows Console APIs are still necessary to establish the initial environment as the Windows platform still differs from others in process, signal, device, and encoding handling:

```

Comment:
=> `limited subset .. is`

Reply:
yes true thanks. tricky phrase in there.